frontend/src/views/ProducerFormPage/ProductStep.vue

```diff
@@ -47,7 +47,11 @@
             <DsfrInput v-model="payload.unitQuantity" class="max-w-64" :required="true" />
           </div>
           <div class="max-w-32 ml-4">
-            <DsfrSelect :options="units" v-model="payload.unitMeasurement" defaultUnselectedText="Unité" />
+            <DsfrSelect
+              :options="units?.map((unit) => ({ text: unit.name, value: unit.name }))"
```

Comment:
Sauf erreur de ma part si le `text` et `value` sont les mêmes, on peu simplement avoir une liste de strings :
```suggestion
              :options="units?.map((unit) => unit.name)"
```

Reply:
fixé ici 49fbf5356b2dcfe5d4ee57eae1891085a505dd05